commcare_connect/static/prelogin/styles.css

```
@@ -489,5913 +647,9728 @@ h3 em {
-.step-detail[data-step='verify'] .step-block-label {
-  color: #fc5f36;
+.service-tab-panel.is-active {
+  display: block;
 }
-.step-detail[data-step='pay'] .step-block-label {
-  color: #1b998b;
+@media (prefers-reduced-motion: no-preference) {
+  .service-tab-panel.is-active {
+    animation: stabFadeIn 0.25s ease both;
+  }
+  @keyframes stabFadeIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "stabFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)